src/experimental/safe_html_transform/safe_html.py

```diff
@@ -0,0 +1,2630 @@
+import logging
```

Comment:
@tisto Oh Sorry!! I took it in worng way :/ .. Did the correction and travis seems to be happy with it ;)